packages/applicant/src/utils/postQuestion.ts

```diff
@@ -250,6 +267,7 @@ export const createRequestBody = (
     submissionId,
     questionId,
     multiResponse: isResponseAnArray ? body[questionId] : multiResponseValues,
+    shouldUpdateSectionStatus: !shouldRedirectToSummary,
```

Comment:
Just to check: this is so we don't set complete sections to in progress when the user edits an answer?

Reply:
yessir